liferay-frontend/liferay-portal
# LPS-102097 - Deactivating compatibility layer for grid

This PR turns off the BS3 compatibility layer for Grid
I've changed for that:

- img old grid clases
- col old clases
- containers old classes

I've split commerce changes in its own commits

---

To test the PR:

- Navigate along portal (especially commerce) and see containers still working fine

---

It is coming from:
https://github.com/liferay-frontend/liferay-portal/pull/497
